src/modules/visitor-space/components/NoServerSideRendering/NoServerSideRendering.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC, ReactNode, useEffect, useState } from 'react';
+
+interface NoSsrProps {
+	children: ReactNode;
+}
+
+export const NoServerSideRendering: FC<NoSsrProps> = ({ children }) => {

```

Comment:
Does this component replace `import { isServerSideRendering } from '@shared/utils'` entirely?
If so, probably should take this opportunity to deprecate / refactor instances of `isServerSideRendering`.

Reply:
no, NoServerSideRendering only for rendering things
the isServerSideRendering() is still usefull for switching in logic not related to rendering